webservices/tasks/__init__.py

```diff
@@ -22,6 +22,10 @@
             'task': 'webservices.tasks.legal_docs.refresh',
             'schedule': crontab(minute='*/5'),
         },
+        'delete_cached_call_folder': {
+            'task': 'webservices.tasks.cache_request.delete_cached_calls_folder',
+            'schedule': crontab(minute=0, hour=10),
```

Comment:
@pkfec should this be `delete_cache_calls_from_s3`?

Reply:
The folder name on S3 is "cached-calls". Thought of giving the same name in my function too

Reply:
The function below https://github.com/fecgov/openFEC/pull/3024/files#diff-8c5fcdb267045bc27a9bfa4825bd19e3R43 has no "d" `delete_cache_calls_folder`